assets/app/views/_triggers.html

```diff
@@ -1,47 +1,49 @@
 <div class="builds-block">
   <div class="builds" ng-repeat="trigger in triggers">
-    <div ng-repeat="build in trigger.builds track by (build | uid)"
-        ng-if="((build | isRecentBuild) || (build | isOscActiveObject)) && !isBuildHidden(build)"
-        class="build animate-repeat osc-object"
-        kind="Build"
-        resource="build">
-      <!-- Icon -->
-      <span ng-switch="build.status.phase" class="hide-ng-leave">
-        <span ng-switch-when="Complete" class="fa fa-check text-success" aria-hidden="true" style="margin-right: 5px;"></span>
-        <span ng-switch-when="Failed" class="fa fa-times text-danger" aria-hidden="true" style="margin-right: 5px;"></span>
-        <span ng-switch-when="Error" class="fa fa-times text-danger" aria-hidden="true" style="margin-right: 5px;"></span>
-        <span ng-switch-when="Cancelled" class="fa fa-ban text-warning" aria-hidden="true"></span>
-        <span ng-switch-when="Pending" class="spinner spinner-xs spinner-inline" aria-hidden="true" style="margin-right: 5px;"></span>
-        <span ng-switch-default class="fa fa-refresh fa-spin" aria-hidden="true" style="margin-right: 5px;"></span>
-      </span>
-      <!-- Message -->
-      Build
-      <span ng-if="build | annotation : 'buildNumber'">
-        <a ng-href="{{build | navigateResourceURL}}">
-          <span ng-if="build.metadata.labels.buildconfig">{{build.metadata.labels.buildconfig}}</span>
-          #{{build | annotation : 'buildNumber'}}
-        </a>
-      </span>
-      <span ng-if="!(build | annotation : 'buildNumber')">
-        {{build.metadata.name}}
-      </span>
-      <span ng-switch="build.status.phase" class="hide-ng-leave">
-        <span ng-switch-when="Complete">completed.</span>
-        <span ng-switch-when="Failed">failed.</span>
-        <span ng-switch-when="Error">encountered an error.</span>
-        <span ng-switch-when="Cancelled">was cancelled.</span>
-        <span ng-switch-default>is {{build.status.phase | lowercase}}.
-          <span ng-if="trigger.imageChangeParams.automatic">
-            A new deployment will be created automatically once the build completes.
+    <div ng-if="trigger.type === 'ImageChange'">
+      <div ng-repeat="build in buildsByOutputImage[(trigger.imageChangeParams.from | imageObjectRef : namespace)] track by (build | uid)"
+          ng-if="((build | isRecentBuild) || (build | isOscActiveObject)) && !isBuildHidden(build)"
+          class="build animate-repeat osc-object"
+          kind="Build"
+          resource="build">
+        <!-- Icon -->
+        <span ng-switch="build.status.phase" class="hide-ng-leave">
```

Comment:
I think you can use the `<status-icon status="build.status.phase">` directive here.